src/main/scala/com/cognite/sdk/scala/v1/dataModels.scala

```diff
@@ -3,81 +3,104 @@
 
 package com.cognite.sdk.scala.v1
 
+final case class SpacedItems[A](spaceExternalId: String, items: Seq[A])
+
+final case class DataModelIdentifier(
+    space: Option[String],
+    model: String
+)
+
+object DataModelIdentifier {
+  def apply(model: String): DataModelIdentifier =
+    DataModelIdentifier(None, model)
+  def apply(space: String, model: String): DataModelIdentifier =
+    DataModelIdentifier(Some(space), model)
+}
```

Comment:
The case class lets us do this (you can pass both `None` and `Some(space)` as space), so unless there's a specific need I think we can just get rid of this one!